source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -148,6 +152,19 @@ Syntax
                       [--airgap]          \
                       ALIAS
 
+.. mc-cmd:: client
+   :fullpath:
+
+   Measure the network throughput from the server to a client.
```

Comment:
Rather than a client, this likely measures perf between the local host at the server, I assume?

Reply:
Clarified this with jiuker.
The client is MinIO Client running on the local machine, measuring the rate to talk to the desired server.

Updated the wording accordingly.